src/main/java/frc/robot/subsystems/drive/SwerveModule.java

```diff
@@ -0,0 +1,166 @@
+package frc.robot.subsystems.drive;
+
+import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import PIDControl.PIDControl;
+import PIDControl.PIDControl.Coefficient;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.AnalogPotentiometer;
+import frc.robot.Constants;
+
+public class SwerveModule extends Vector
+{
+    //private CANSparkMax          _driveMotor;
+    private TalonFX                _driveMotor;
+    private CANSparkMax          _rotateMotor;
+    private AnalogPotentiometer  _rotateSensor;
+    private PIDControl           _rotatePID;
+    
+    private double            _rotateSetpoint;
+    private double            _driveSpeedSetpoint;
+
+    private double            _rotationZero;
+    private double            _resetOffset;
+
+    private double            _drivePosition;
+
+    public SwerveModule(double x,// (positive right) relative to the center of the robot 
+                        double y,// (positive forward) relative to the center of the robot
+                        double defaultRelativeZero,// default zeroing position
+                        double resetOffset,// for physically rotated modules, reset the offsets at a different angle
```

Comment:
The comments are helpful, but super cramped. Can we align them farther to the left to make this easier to read?

```suggestion
    public SwerveModule(double x,                      // (positive right) relative to the center of the robot 
                        double y,                      // (positive forward) relative to the center of the robot
                        double defaultRelativeZero,    // default zeroing position
                        double resetOffset,            // for physically rotated modules, reset the offsets at a different angle
```